src/openrct2/interface/FontFamilies.cpp

```diff
@@ -0,0 +1,44 @@
+#pragma region Copyright(c) 2014 - 2017 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "FontFamilies.h"
+
+#ifndef NO_TTF
+
+TTFontFamily const TTFFamilyChineseSimplified = {
```

Comment:
Braces go on their own lines.

Reply:
Ah. For blocks, I get that, but for initialisers, too? I must say that feels counterintuitive to me. How would you like me to apply this w.r.t. the definitions for e.g. `TTFFontHiragano` in FontFamilies.cpp?